2.train_model/README.md

```diff
@@ -1,10 +1,11 @@
 # 2. Train Model
 
-In this module, we train a ML model to predict phenotypic class from DeepProfiler-extracted features.
+In this module, we train ML models to predict phenotypic class from cell features.
 
-We train the model in [train_model.ipynb](1.train_model.ipynb).
-We use [sklearn.linear_model.LogisticRegression](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html) for our machine learning model.
-We use the following parameters for our Logisic Regression model:
+We train the models in [train_model.ipynb](1.train_model.ipynb).
+We use [sklearn.linear_model.LogisticRegression](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html) for our machine learning models.
+A final model 
```

Comment:
this looks like an incomplete sentence or typo?

Reply:
A typo, fixed in https://github.com/WayScience/phenotypic_profiling_model/pull/20/commits/c2a67f2d3c048099a41dcf4ad606c374181af490